lib/Sema/ConstraintSystem.h

```diff
@@ -2159,6 +2159,11 @@ class ConstraintSystem {
                    bool skipProtocolSelfConstraint,
                    ConstraintLocatorBuilder locator,
                    OpenedTypeMap &replacements);
+  void openGenericRequirements(DeclContext *outerDC,
```

Comment:
Can you add a comment explaining what this does?

Reply:
Done!